src/xvectorbin/nnet3-xvector-signal-perturb-egs.cc

```diff
@@ -51,8 +51,8 @@ void ApplyPerturbation(XvectorPerturbOptions opts,
     // randomly generate an zero-phase FIR filter with no zeros.
     // In future, we can select trucated part of room impluse response
     // and convolve it with input_egs.
-    perturb_xvector.ComputeAndApplyRandDistortion(shifted_egs,
-                                  &rand_distort_shifted_egs);
+    ////perturb_xvector.ComputeAndApplyRandDistortion(shifted_egs,
+    ////                              &rand_distort_shifted_egs);
```

Comment:
remove ApplyPerturbation function from nnet3-perturb-egs. We will process egs in distort-signal.*